ocaml/xapi/taskHelper.ml

```diff
@@ -53,6 +53,17 @@ let rbac_assert_permission_fn = ref None
 
 (* required to break dep-cycle with rbac.ml *)
 
+let compare_auth_user_ids_of_sessions ~__context s1 s2 =
```

Comment:
This returns a boolean but `compare` suggests it returns an order (-1,0,1). So this should have `equal` in its name.

Reply:
Thanks! I updated it :)